### PR TITLE
Add Service Account and roles for integration

### PR DESCRIPTION
**Pull request by janboll**
### What this PR does
https://issues.redhat.com/browse/ARO-9988

Jira: 
Link to demo recording:

### CODE REVIEW

File: cluster-service/deploy/integration/cluster-service-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      namespace: cluster-service-admin
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: namespace-admin
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - namespace
```
Comment (oriAdler) [inline cluster-service/deploy/integration/cluster-service-namespace.yaml:24]: Does the cluster role need to include: serviceaccounts and tokens?

File: cluster-service/deploy/integration/cluster-service-namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      kind: ClusterRole
+      name: admin
+    subjects:
+    - kind: ServiceAccount
+      name: cluster-service-mgmt
+      namespace: cluster-service-admin
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: cluster-service-naemspace-admin
```
Comment (oriAdler) [inline cluster-service/deploy/integration/cluster-service-namespace.yaml:42]:
```suggestion
      name: cluster-service-namespace-admin
```

File: cluster-service/deploy/integration/cluster-service-namespace.yaml (hunk 1 above)
Comment (oriAdler) [reply cluster-service/deploy/integration/cluster-service-namespace.yaml:24]: I guess `cluster-admin` would be too much for our case, or given that it's a test environment, is it reasonable?

Comment (janboll) [reply cluster-service/deploy/integration/cluster-service-namespace.yaml:24]: Tried it locally using minikube with 1.31, there it was not included in admin

Comment (janboll) [reply cluster-service/deploy/integration/cluster-service-namespace.yaml:24]: I think clusteradmin would be to wide

Comment (oriAdler) [reply cluster-service/deploy/integration/cluster-service-namespace.yaml:24]: Thanks for the clarification:)

**Pull request merged**